source/reference/mongosync-behavior.txt

```diff
@@ -21,8 +19,13 @@ The ``mongosync`` binary is the primary process used in
 {+c2c-product-name+}. ``mongosync`` migrates data from one cluster to
 another and can keep the clusters in continuous sync. 
 
-This document provides a complete overview of the ``mongosync`` command
-line options.
+For information on how the ``mongosync`` process works, see 
+:ref:`about-mongosync`. 
+
+To get started with ``mongosync``, refer to the :ref:`Quick Start Guide
+<c2c-quickstart>`. For more detailed information, refer to the
+:ref:`c2c-install` or :ref:`c2c-connecting` page that best fits your
+situation. 
```

Comment:
[suggestion] I would suggest splitting this up to be consistent with the space between the first more information line (that links to about mongosync) and these lines

```suggestion
To get started with ``mongosync``, refer to the :ref:`Quick Start Guide
<c2c-quickstart>`. 

For more detailed information, refer to the
:ref:`c2c-install` or :ref:`c2c-connecting` page that best fits your
situation. 
```

Reply:
I like this, fixing now